bitnami/multus-cni/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRoleBinding
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+metadata:
+  name: {{ include "common.names.fullname.namespace" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: server
```

Comment:
Typo? `component` label is not defined in the rest of the kubernetes objects

Reply:
Fixed